scripts/west_commands/sign.py

```diff
@@ -81,18 +81,18 @@
 '''
 
 
-def config_get_words(west_config, section_key, fallback=None):
-    unparsed = west_config.get(section_key)
-    log.dbg(f'west config {section_key}={unparsed}')
+def config_get_words(cmd, section_key, fallback=None):
+    unparsed = cmd.config.get(section_key)
+    cmd.dbg(f'west config {section_key}={unparsed}')
     return fallback if unparsed is None else shlex.split(unparsed)
 
 
-def config_get(west_config, section_key, fallback=None):
-    words = config_get_words(west_config, section_key)
+def config_get(cmd, section_key, fallback=None):
+    words = config_get_words(cmd, section_key)
     if words is None:
         return fallback
     if len(words) != 1:
-        log.die(f'Single word expected for: {section_key}={words}. Use quotes?')
+        cmd.die(f'Single word expected for: {section_key}={words}. Use quotes?')
```

Comment:
Actually, `cmd` really looks like `self` now... would it make sense to move these two and make them `Forceable` methods now? If you agree but don't have time to do it, add a `TODO` suggestion in a comment?

Reply:
It does make sense, I'll update.

Reply:
I meant even higher to the base class, see #79678 that I just submitted.

Reply:
> see https://github.com/zephyrproject-rtos/zephyr/pull/79678 that I just tested and submitted.

Merged, git conflict now, need to rebase (upvote https://github.com/orgs/community/discussions/4335 )